docs/resources/cluster_gke.md

```diff
@@ -0,0 +1,301 @@
+---
+page_title: "spectrocloud_cluster_gke Resource - terraform-provider-spectrocloud"
+subcategory: ""
+description: |-
+  Resource for managing GKE clusters in Spectro Cloud through Palette.
```

Comment:
GKE clusters through Palette - should be good

Reply:
done